nms/app/packages/magmalte/alerts/lteAlerts.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import type {prom_alert_config} from '@fbcnms/magma-api';
+
+export const lteAlerts: {[string]: prom_alert_config} = {
+  'Service Restart Alert': {
+    alert: 'Service Restart Alert',
+    expr: 'increase(unexpected_service_restarts[5m]) > 0',
+    labels: {severity: 'minor'},
+    annotations: {description: 'Alerts upon service restarts'},
+  },
+  'Cpu Percent Alert': {
+    alert: 'Cpu Percent Alert',
+    expr: 'increase(cpu_percent[5m]) > 75',
```

Comment:
this is a gauge so you don't want `increase()`. You should do an `avg_over_time(cpu_percent[5m]) > 75` instead